examples/static-site/Caddyfile

```diff
@@ -0,0 +1,5 @@
+0.0.0.0:8080 {
+        root /var/www/public
+        log stdout
+        gzip
```

Comment:
how could it be done? Avoiding the compression of images having certain extensions like .jpg or .png (for example)? Would it be useful?